relational_operators/tests/SortMergeRunOperator_unittest.cpp

```diff
@@ -1404,13 +1407,14 @@ class SortMergeRunOperatorTest : public ::testing::Test {
     std::size_t num_receieved = 0;
     do {
       if (bus_.ReceiveIfAvailable(foreman_client_id_, &msg)) {
-        EXPECT_EQ(kWorkOrderFeedbackMessage, msg.tagged_message.message_type());
-        WorkOrder::FeedbackMessage feedback_msg(
-            const_cast<void *>(msg.tagged_message.message()),
-            msg.tagged_message.message_bytes());
-        EXPECT_EQ(kOpIndex, feedback_msg.header().rel_op_index);
-        merge_op_->receiveFeedbackMessage(feedback_msg);
-        num_receieved++;
+        if (msg.tagged_message.message_type() == kWorkOrderFeedbackMessage) {
```

Comment:
Please ignore my previous comment. 

Earlier the TMB client registration was not properly done, therefore TMB didn't receive any pipelining messages. Because of the new checks now pipelining messages are also being sent. However in the test, it is not required to process the pipelining messages. The reason is that the function is written to check and dispatch only feedback messages and hence the if condition.